src/pr-gating/CIGate.ps1

```diff
@@ -0,0 +1,74 @@
+# This script is used as part of our PR gating strategy. It takes advantage of the GHAzDO REST API to check for CodeQL issues a PR source and target branch. 
+# If there are new issues in the source branch, the script will report on that and fail with error code 1. 
+$pass = ${env:MAPPED_ADO_PAT}
+$orgUri = ${env:SYSTEM_COLLECTIONURI}
+$orgName = $orgUri -replace "^https://dev.azure.com/|/$"
+$project = ${env:SYSTEM_TEAMPROJECT} 
+$repositoryId = ${env:BUILD_REPOSITORY_ID} 
+$prTargetBranch = ${env:SYSTEM_PULLREQUEST_TARGETBRANCH}
+$prSourceBranch = ${env:BUILD_SOURCEBRANCH}
+$pair = ":${pass}"
+$bytes = [System.Text.Encoding]::ASCII.GetBytes($pair)
+$base64 = [System.Convert]::ToBase64String($bytes)
+$basicAuthValue = "Basic $base64"
+$headers = @{ Authorization = $basicAuthValue }
+
+$urlTargetAlerts = "https://advsec.dev.azure.com/{0}/{1}/_apis/Alert/Repositories/{2}/Alerts?top=5000&orderBy=lastSeen&criteria.alertType=3&criteria.branchName={3}&criteria.onlyDefaultBranchAlerts=true&useDatabaseProvider=true" -f $orgName, $project, $repositoryId, $prTargetBranch
```

Comment:
This script appeared broken for me ... 

I updated these in the commit: [Fix Alert URLs](https://github.com/microsoft/GHAzDO-Resources/pull/14/commits/d166721eadf2f6a5e590e4100a7819bd55b475f6)

Reply:
Yes, correct. I messed up in copying... Thanks for correcting.